cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py

```diff
@@ -1207,10 +1207,22 @@ def test_successful_update_request(
     def test_dryrun(self, mocker, client):
```

Comment:
nit: shall we also test the suppressed validation messages? Aren't we testing dryrun for create?

Reply:
https://github.com/aws/aws-parallelcluster/pull/3122/commits/61fdaeae8c93895f9f0d9949604ac54ecb292d64